README.md

```diff
@@ -23,16 +23,16 @@ $ pip install --upgrade pip && pip install hyperdash
 Installing within a python virtual environment such as [virtualenv](https://github.com/pypa/virtualenv) is recommended. If you are having trouble installing via pip, a virtual environment will usually fix the problem.
```

Comment:
We may want to explicitly state that users get `hd` when they install `hyperdash`. Additionally, existing users are going to wonder why the change.

Reply:
Yeah I can make that more explicit. Also 'hyperdash' will still work